Plugins/InjectGeneratorPlugin/Plugin.swift

```diff
@@ -1,8 +1,15 @@
 import PackagePlugin
 import Foundation
 
-private func readIncludefile(workingDirectory: URL) throws -> [String] {
-    let includefileURL = workingDirectory.appendingPathComponent(".inject").appendingPathComponent("Includefile")
+private func readIncludefile(workingDirectory: URL) -> [String] {
+    let injectDirectory = workingDirectory.appendingPathComponent(".inject")
+    let includefileURL = injectDirectory.appendingPathComponent("Includefile")
+    let fileManager = FileManager.default
+    // Ensure both .inject directory and Includefile exist
+    guard fileManager.fileExists(atPath: injectDirectory.path),
+          fileManager.fileExists(atPath: includefileURL.path) else {
+        return []
+    }
     guard let content = try? String(contentsOf: includefileURL, encoding: .utf8) else {
```

Comment:
As it tries optionally to read content file, if file does not exist it simply fails and goes to guard's else where and empty array returns. So I think this change is not necessary.